common/train.py

```diff
@@ -28,6 +28,7 @@ class TrainerFactory(object):
         'Reuters': ReutersTrainer,
         'AAPD': ReutersTrainer,
         'IMDB': ReutersTrainer,
+        'Yelp2014': ReutersTrainer,
```

Comment:
Inconsistent name.

Reply:
Can you please elaborate on this? Do you suggest we rename ReutersTrainer to GenericDocTrainer or something?

Reply:
Yeah sure.